packages/react/src/components/dateInput/DateInput.test.tsx

```diff
@@ -296,4 +296,18 @@ describe('<DateInput /> spec', () => {
     // No date should be selected in the calendar
     expect(container.querySelector('[aria-pressed="true"]')).toBeNull();
   });
+
+  it('should be able to clear the value', async () => {
+    const { rerender } = render(<DateInput id="date" label="Foo" value="10.02.2022" />);
+
+    // The initial value should be there
+    expect(screen.getByRole('textbox')).toHaveValue('10.02.2022');
```

Comment:
The test could also check that DatePicker shows the value's month:
```js
expect(screen.getByLabelText('Month')).toHaveValue('1');
expect(screen.getByLabelText('Year')).toHaveValue('2022');
```
  
and then shows today's (mocked today is 17.1.2021 in this testset) month:
```js
expect(screen.getByLabelText('Month')).toHaveValue('0');
expect(screen.getByLabelText('Year')).toHaveValue('2021');
```

Reply:
Added